test/test-ce.html

```diff
@@ -8,7 +8,11 @@
     <script src="/src/fragment-ce.js" async></script>
   </head>
   <body>
+    <x-fragment config>
+      <config header cache-control="must-revalidate"></config>
+    </x-fragment>
```

Comment:
Sounds good. Then, we should bear in mind that ```header-``` is an object.
Said that, let's assume that ```header-``` will be a CONST that we'll wire up with options.
Also, I'm wondering... what about if define a param as:
```html
<meta x-fragment headers-cache-control="must-revalidate">
```

Should we encourage to use __plural__ form?